plugins/chat/app/controllers/api_controller.rb

```diff
@@ -12,4 +12,18 @@ def ensure_can_chat
     raise Discourse::NotFound unless SiteSetting.chat_enabled
     guardian.ensure_can_chat!
   end
+
+  def default_actions_for_service
```

Comment:
Moved these default actions in `ApiController` since they’re controllers specific.

Reply:
This should probably be in `Chat::ChatBaseController` instead since that is used for the other non-API controllers. But it's a good move :+1:

Reply:
These default actions are clearly API oriented though, so not sure we should move them in the more generic `Chat::ChatBaseController`?

Reply:
How are they API oriented? It's just all controller stuff.

Reply:
These default responses make sense for _any_ controller not just out chat API controllers.

Reply:
Rendering JSON is API oriented. More “classic” controllers could render a view for example :slightly_smiling_face:

Reply:
I mean, sure, but almost every single one of our controllers (not just in chat) does a JSON response so I think this is just hair-splitting...in reality this could be used in any of our controllers. Anyway we've just put it into the API one for now, we can move it later if needed.